### ISSUE TITLE
[a11y] Toolbar url bar doesn't announce it's editable

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-echo-show#46]
https://github.com/mozilla-mobile-skunkworks/firefox-connect/issues/382
